portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +27,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private List<String> comments = new ArrayList<>();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(comments);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
-  }
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment");
-    comments.add(text);
+    long time = System.currentTimeMillis();
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("comment", text);
```

Comment:
Generally, it is good practice to not hard-code constants like this in the code. Recommended approach is to define these as static constants. Basically, define them at the top of this class, as:

```java
private static final String COMMENT_ENTITY_KEY = "Comment";
private static final String COMMENT_PROPERTY_NAME = "comment";
private static final String TIME_PROPERTY_NAME = "time";
```

This makes the code more readable.



Optional if you want to make the change.